content/docs/user-guide/dvc-files.md

```diff
@@ -118,14 +118,14 @@ and `dvc commit` commands, but not when a `.dvc` file is overwritten by
 
 ## `dvc.yaml` file
 
-`dvc.yaml` files describe data science or machine learning pipelines, similar to
+`dvc.yaml` files describe data science or machine learning pipelines (similar to
 how [Makefiles](https://www.gnu.org/software/make/manual/make.html#Introduction)
-work for building software. Its YAML structure contains a list of stages which
+work for building software). Its YAML structure contains a list of stages which
```

Comment:
@shcheklein FYI some of the first few changes in this file I missed moving to the misc. updates PR we just merged, sorry! They're just small copy edits...